tedana/selection/selection_utils.py

```diff
@@ -253,6 +253,27 @@ def comptable_classification_changer(
                             "accepted or rejected, it shouldn't be reclassified"
                         )
             selector.component_table_.loc[changeidx, "classification"] = classify_if
+
+        if tag_if is not None:  # only run if a tag is provided
+            # if tag_if has commas, divide into multiple tags
+            if "," in tag_if:
+                multi_tags = tag_if.split(",")
+                tag_if = set([s.strip() for s in multi_tags])
+            else:
+                tag_if = {tag_if}
```

Comment:
Is this necessary? If the input is a comma-delimited string and the field in the component table is a comma-delimited string, do you need to modify it?

Reply:
This makes sure the same tag isn't repeated. That is, if the existing table has `tag1,tag2` and the new tags are `tag2,tag3` the result will be `tag1,tag2,tag3` without a repetitionof `tag2`

Reply:
Ah okay, thanks. I didn't realize this happened after the old and new tags were concatenated.